packages/world/src/modules/core/implementations/WorldRegistrationSystem.sol

```diff
@@ -262,6 +262,13 @@ contract WorldRegistrationSystem is System, IWorldErrors {
     }
   }
 
+  function unregisterDelegation(address delegatee) public {
+    // Delete the delegation control contract address
+    UserDelegationControl.deleteRecord({ delegator: _msgSender(), delegatee: delegatee });
+
+    // TODO: clear the specific delegation control
```

Comment:
Do we care about deleting the entries in the `DelegationControl`. If so should they implement some `deleteDelegation` function? 

Semantically, it's not obvious that calling `initDelegation` with empty calldata should "uninitialize" the delegation. For example, the control might increment some counter each time a delegation is registered.